prebid-server/endpoints/openrtb2/video.md

```diff
@@ -0,0 +1,355 @@
+---
+layout: page_v2
+sidebarType: 5
+title: Prebid Server | Endpoints | Video
+
+---
+
+# Prebid Server Video Endpoint
+{: .no_toc }
+This document describes the behavior of the Prebid Server `video` endpoint in detail.
+
+* TOC
+{:toc}
+
+## Video Endpoint
+ `POST /openrtb2/video`
+
+
+
+## Overview
+
+Prebid Server (PBS) supports competitve separation for ad pod display from primary ad servers such as Freewheel and DFP.  PBS, on receiving this endpoint, will pass default settings and `stored_request` parameters to one or more OpenRTB requests, which will then be sent to the appropriate SSPs. 
+
+The process is similar to [Prebid.js client side](/prebid-video/video-long-form.html) support for this feature.
+
+### Process
+
+1. Application makes request for a video stream. 
+2. An SSAI Server sends an OpenRTB request to PBS.
+3. PBS sends a request for bids to selected demand partners by sending the OpenRTB request to them. 
+4. Demand partners return a bid response to PBS. If competitive seperation is enabled, PBS peforms [category translation](/dev-docs/modules/categoryTranslation.html) on each bid. Whether category translation is required or not, the bids are stored in prebid cache. 
+5. PBS generates key-value pairs that are comprised of price, industry and duration values. The key is `hb_pb_industry_duration` and each component of the key after the `hb` represents a related value.    
```

Comment:
The key name prebid server uses is `hb_pb_cat_dur` for price bucket, category, duration.

Reply:
Fixed.